magento/graphql-ce
# Shopping cart grand total and taxes coverage

### Description (*)
This PR introduces a possibility to view cart totals as well as taxes applied within scope of the current shopping cart

### Fixed Issues (if relevant)
1. #426: [Place order] Cart tax
2. #425: [Place order] Cart grand total

### Manual testing scenarios (*)
1. Configure taxes for products/zones/customer
2. Add a taxable product to the shopping cart
3. Assign shipping address (that corresponds a taxable zone) to the shopping cart
4. Use the following query to check the totals and taxes:

```graphql
{
  cart(cart_id: "$cart_id") {
    prices {
      grand_total {
        value,
        currency
      }
      subtotal_including_tax {
        value
        currency
      }
      subtotal_excluding_tax {
        value
        currency
      }
      subtotal_with_discount_excluding_tax {
        value
        currency
      }
      applied_taxes {
        label
        amount {
          value
          currency
        }
      }
    }
  }
}
```
